app/src/main/res/values/integer.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    // CreateProfileFragment
+    <integer name="mt_username_min_length">4</integer>
```

Comment:
Is there a reason for defining these constants in an xml?

Reply:
I prefer to have everything at one place but I guess just my preference